sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2634,6 +2643,48 @@ SIREPO.app.directive('bootstrapToggle', function() {
     };
 });
 
+
+SIREPO.app.directive('jobsList', function(requestSender, appState) {
+    return {
+        restrict: 'A',
+        template: [
+            '<div>',
+                '<table class="table">',
+                '<tr>',
+                    '<th data-ng-repeat="c in data.columns">{{ c }}</th>',
+                '</tr>',
+                '<tr data-ng-repeat="r in data.data">',
+                    // must 'track by $index' because start and last update can be the same
+                    '<td data-ng-repeat="c in r track by $index">',
+                        '<span>{{ c }}</span>',
+                    '</td>',
+                '</tr>',
+                '</table>',
+                '<button class="btn btn-default" data-ng-click="getAdmJobs()">Refresh</button>',
+            '</div>',
+        ].join(''),
+        controller: function($scope) {
+            function dataLoaded(data, status) {
+                $scope.data = data;
+            };
+
+            $scope.getAdmJobs = function (id) {
+                requestSender.sendRequest(
+                    'admJobs',
+                    dataLoaded,
+                    {
+                        id: id,
+                    });
+            };
+
+            appState.clearModels(appState.clone(SIREPO.appDefaultSimulationValues));
```

Comment:
@moellep I've added this piece of code because I want all the header values (ex sectionTitle link, source tab, etc) to not be shown on the adm jobs list page (see image below). It looked like the [simulationsController was doing the same thing](https://github.com/radiasoft/sirepo/blob/master/sirepo/package_data/static/js/sirepo.js#L3295) when a user navigates to the simulations page. Does this look right? Could calling this / not calling something else break something I'm unaware of?

![Screen Shot 2020-02-28 at 11 00 30 AM](https://user-images.githubusercontent.com/10264515/75573264-b5437280-5a19-11ea-8c23-eea33cdfbeb0.png)

Reply:
Yes, this is fine for now - any models should get cleared out when someone views the jobs page, similar to the simulation list page. Really the SIREPO.appDefaultSimulationValues should access default-data.json for the simulation, but that can wait.